docs/change-streams.md

```diff
@@ -1,7 +1,7 @@
 ## Change Streams
 
 [Change streams](https://www.mongodb.com/developer/quickstart/nodejs-change-streams-triggers/) let you listen for updates to documents in a given model's collection, or even documents in an entire database.
-Unlike [middleware](/docs/middleware.html), change streams are a MongoDB server construct, which means they pick up changes from anywhere.
+Unlike [middleware](../docs/middleware.html), change streams are a MongoDB server construct, which means they pick up changes from anywhere.
```

Comment:
I may be missing something, but why not just do `./middleware.html` ? The `../docs` doesn't seem to be necessary.

Reply:
there were many places where this would apply, and i "overlooked" it because this PR is basically not modifying the link but making them relative from how they were before, maybe with #12645 we could change them to be more relative?

Reply:
Let me know what y'all want, I can make the related change